### PR TITLE
Python hooks

### PR DESCRIPTION
**Pull request by tchellomello**
This PR adds a hook examples written in Python.

### CODE REVIEW

File: examples/python/host/managed/after_create/12-ppc64-wrapper.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
```
Comment (domcleal) [inline examples/python/host/managed/after_create/12-ppc64-wrapper.py]: This is out of scope for foreman_hooks, I'm not able to review or maintain it - please publish it to a different repository.

File: examples/python/hook_functions.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+#######
+# To allow this module to be imported by other triggers
+# please place this file at /usr/share/foreman/config/
```
Comment (domcleal) [inline examples/python/hook_functions.py]: This is not a general purpose directory, it's part of the Foreman application config - the recommendation should be elsewhere, or preferably in system-wide Python module dir.

File: examples/python/host/managed/after_create/12-ppc64-wrapper.py (hunk 1 above)
Comment (nvtkaszpir) [inline examples/python/host/managed/after_create/12-ppc64-wrapper.py]: looks like maybe community-hooks repo? something like community-templates?

**Comment by tchellomello** [comment]
@domcleal directory changed. Please let me know if you anything else is needed.

**Comment by domcleal** [comment]
Could you also remove the extended examples please? I'd prefer to only ship a basic one, I don't want to be handling PRs for the specific examples (like PPC, MS DNS etc).

**Comment by tchellomello** [comment]
@domcleal  PR updated. I understand your point but it would be nice to have an area on this repository (or another) where the community would keep the scripts and other examples using foreman hooks. 

Thanks @domcleal

**Comment by domcleal** [comment]
Merged, thanks @tchellomello. I don't wish to collect or maintain any scripts within this repo beyond very basic per-language outlines, but please do publish yours on GitHub or similar for others to use.
